sdk/keyvault/keyvault-certificates/samples/mergeCertificate.ts

```diff
@@ -0,0 +1,47 @@
+import { CertificatesClient } from "../src";
+import { EnvironmentCredential } from "@azure/identity";
+
+export function delay<T>(t: number, value?: T): Promise<T> {
+  return new Promise((resolve) => setTimeout(() => resolve(value), t));
+}
+
+async function main(): Promise<void> {
+  // EnvironmentCredential expects the following three environment variables:
+  // - AZURE_TENANT_ID: The tenant ID in Azure Active Directory
+  // - AZURE_CLIENT_ID: The application (client) ID registered in the AAD tenant
+  // - AZURE_CLIENT_SECRET: The client secret for the registered application
+  const vaultName = process.env["KEYVAULT_NAME"] || "<keyvault-name>"
+  const url = `https://${vaultName}.vault.azure.net`;
+  const credential = new EnvironmentCredential();
```

Comment:
DefaultAzureCredential is a superset of EvironmentCredential I believe, and would make the sample "just work" in MSI scenarios. Might be worth using that for samples instead?